blog: introducing localstack extension for docker desktop (#73)
localstack/localstack.github.io: content/blog/2022-10-25-introducing-localstack-extension-for-docker-desktop/index.md

---
title: Introducing LocalStack Extension for Docker Desktop
description: Introducing LocalStack Extension for Docker Desktop
lead: Introducing LocalStack Extension for Docker Desktop
date: 2022-10-25T4:18:20+05:30
lastmod: 2022-10-25T4:18:20+05:30
images: []
contributors: ["Harsh Mishra"]
tags: ['news']
leadimage: "header.png"
---

> We are excited to announce the LocalStack Extension for Docker Desktop, which helps developers to run their cloud & serverless applications on Amazon Web Services (AWS) locally. The LocalStack Extension for Docker Desktop provides users with an integral, fully-integrated experience to manage their LocalStack instance and configurations and check the logs. The extension aims to bridge the user experience for developers who want to use LocalStack more effectively!

Reviewer: ```suggestion
We are excited to announce the LocalStack Extension for Docker Desktop, which helps developers to run their cloud & serverless applications on Amazon Web Services (AWS) locally. The LocalStack Extension for Docker Desktop provides users with a fully-integrated experience to manage their LocalStack instance, maintain different configurations, check the logs, and more. The extension aims to bridge the user experience for developers who want to use LocalStack more effectively!
```

## Getting started

> LocalStack is an easy-to-use test/mocking framework for developing cloud applications. Using LocalStack, you can spin up a local test environment in seconds and get the same functionality you would get from a real AWS environment. LocalStack provides [core cloud emulation](), [team collaboration](), and [enterprise features]() to help developers get back control over their environments and free them from wasting time with inefficient dev & test loops in the cloud.

Reviewer: ```suggestion
LocalStack is an easy-to-use cloud emulation platform for developing cloud applications locally. Using LocalStack, you can spin up a local test environment in seconds and get the same functionality you would get from a real AWS environment. LocalStack provides [core cloud emulation](), [team collaboration](), and [enterprise features]() to help developers get back control over their environments and free them from wasting time with inefficient dev & test loops in the cloud.
```

Some of the key features of the LocalStack Extension for Docker Desktop include:

* **Control LocalStack**: Start, stop, and restart LocalStack from the Docker Desktop. You can also see the current status of your LocalStack instance and navigate to LocalStack Web Application.
* **LocalStack insights**: You can see the log information of the LocalStack instance and all the available services and their status on the service page.
* **LocalStack configurations**: You can manage and use your profiles via configurations and create new configurations for your LocalStack instance.

Before the LocalStack Extension for Docker Desktop, we shipped [LocalStack Cockpit](), a fully-integrated desktop client for our users. On the other hand, the LocalStack Extension for Docker Desktop aims to support the community that backs Docker Desktop to get started with managing LocalStack without any fuss! The LocalStack Extension for Docker Desktop is a community project right now and can be [installed via our Docker image]().

## What's next?

> We have released an initial version of the LocalStack Extension for Docker Desktop, and we will continue adding more features to it to make your LocalStack workflow as smooth as possible. Our upcoming plans would be to get you more visibility inside your LocalStack instance runs, enable more excellent compatibility with our [LocalStack Web App](), and make it easy for you to develop & test AWS apps with the power of Docker Desktop Extensions.

Reviewer: ```suggestion
We have released an initial version of the LocalStack Extension for Docker Desktop, and we will continue adding more features to it to make your LocalStack workflow as smooth as possible. Our upcoming plans would be to get you more visibility inside your LocalStack instance runs, enable more seamless compatibility with our [LocalStack Web App](), and make it easy for you to develop & test AWS apps with the power of Docker Desktop Extensions.
```
